utils/src/main/scala/za/co/absa/hermes/utils/FileReader.scala

```diff
@@ -0,0 +1,37 @@
+package za.co.absa.hermes.utils
+
+import scala.io.Source
+
+object FileReader {
+  def readFileAsListOfLines(filename: String): List[String] = {
+    val sourceFile = Source.fromFile(filename)
+    try {
+      sourceFile.getLines().toList // making it a List to copy the content of the file into memory before it's closed
+    } finally {
+      sourceFile.close()
+    }
+  }
+
+  def readFileAsString(filename: String, lineSeparator: String = "\n"): String = {
+    val sourceFile = Source.fromFile(filename)
```

Comment:
__Suggestion__:
You can find Automatic Resource Management pattern useful in this case:
E.g. by using this simple util https://github.com/AbsaOSS/spline/blob/develop/commons/src/main/scala/za/co/absa/spline/common/ARM.scala
```scala
ARM.using(Source.fromFile(filename))(_.getLines.mkString(lineSeparator))
```

Reply:
I think I have this on line 34. `FileReader.usingFile`. 
```
FileReader.usingFile(Source.fromURL(path)) { _.getLines().toList }
```

But I could reuse this method for the mention ones.